compressible_test.go

```diff
@@ -45,7 +45,7 @@ func BenchmarkEstimate(b *testing.B) {
 		b.Log(Estimate(testData))
 	})
 
-	// (not predictable, high entropy distibution)
+	// (not predictable, high entropy length)
```

Comment:
You started replacing with length here for some reason.

Reply:
> You started replacing with length here for some reason.

Sorry, that wasn't on purpose. This is fixed now.